__tests__/app.test.js

```diff
@@ -376,6 +376,87 @@ describe("10. GET /api/reviews/:review_id (comment count)", () => {
   });
 });
 
+describe("11. GET /api/reviews (queries)", () => {
+  test("end point should be able to accept category query that filters the response to only the given category", () => {
+    return request(app)
+      .get("/api/reviews?category=euro game")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(1);
+        body.forEach((review) => {
+          expect(review).toMatchObject({
+            title: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_img_url: expect.any(String),
+            category: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });
+  test("end point should be able to accept category query that filters the response to only the given sort_by", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body[0].review_id).toBe(12);
```

Comment:
Ideally, you should be using jest-sorted here rather than just asserting against the first result.